lib/api/controller/auth.js

```diff
@@ -188,9 +189,74 @@ class AuthController extends NativeController {
         request.context.token);
     }
 
+    if ( global.kuzzle.config.security.supportCookieAuthentication
+      && request.input.args.cookieOnly
+    ) {
+      request.setResult(null,
+        {
+          headers: {
+            'Set-Cookie': Cookie.serialize(
+              '__Host-authToken',
+              '',
+              {
+                httpOnly: true,
+                path: '/',
+                sameSite:'none',
+                secure: true,
+              }
+            )
+          }
+        });
+    }
+
     return {acknowledged: true};
   }
 
+  // Used to send the Token using different ways when in cookieOnly mode. (DRY)
+  async _sendToken(token, request) {
+    // Only if the support of Browser Cookie as Jwt is enabled
+    // otherwise we should send a normal response because
+    // even if the SDK / Browser can handle the cookie,
+    // Kuzzle would not be capable of doing anything with it
+    if (global.kuzzle.config.security.supportCookieAuthentication
```

Comment:
Nitpicking: coding standards
```suggestion
    if ( global.kuzzle.config.security.supportCookieAuthentication
```